### ISSUE TITLE
Expects Vue to be on window

### ISSUE THREAD

**Issue by petridw** [dperrymorrow/electron-vue-debugger#4]
https://github.com/dperrymorrow/electron-vue-debugger/blob/eb1fe915278c1cd6c5600f552346e77f92091fed/src/NavTree.vue#L38

Looks like this expects Vue to be on the window, if it's not it bombs out here. Should be pretty easy to change it to use `this.$set` instead or `import Vue from "vue"` at top of file.

**Comment by dperrymorrow** [comment]
like this? 7095341de860d71e3e3fec18dc60aac3e85d0885

**Comment by petridw** [comment]
yeah p much. not sure if there are anymore global references but that was the only one I saw at first
